libr/core/cmd_anal.c

```diff
@@ -3873,6 +3866,7 @@ static int cmd_anal_all(RCore *core, const char *input) {
 		"aat", " [len]", "analyze all consecutive functions in section",
 		"aap", "", "find and analyze function preludes",
 		"aav", " [sat]", "find values referencing a specific section or map",
+		"aau", " [len]", "list mem areas (larger than len bytes) not covered by functions",
```

Comment:
I agree.. 
but problem is, now **ai** is used to print information about an address (page permissions, etc.)
which btw should probably not be under *a* since it is not related to analysis.